### PR TITLE
Added kdbindings/1.0.3

### PR DESCRIPTION
**Pull request by bog-dan-ro**
**kdbindings/1.0.3**

KDBindings enables reactive programming & data binding in C++ From plain C++ you get:
- Signals + Slots.
- Properties templated on the contained type.
- Property bindings allowing reactive code to be written without having to do all the low-level, error prone plumbing by hand. Lazy evaluation of property bindings.
- No more broken bindings.
- Totally stand-alone "header-only" library.

I'm working for KDAB (the company behind this library). I have their support for adding and maintaining this library to conan center index.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by RubenRBS — APPROVED** [review]

**Comment by RubenRBS** [comment]
Thanks a lot! We appreciate it, the recipe looks good :)

**Review by czoido — APPROVED** [review]

**Comment by RubenRBS** [comment]
@bog-dan-ro nice to hear :)

There's actually no requirement for header-only libraries to define _any_ setting. You're in fact encouraged not to define the attribute at all if the `generate()` method does not use its values/invokes any compiler, etc.
But if you do need them, adding them all usually avoids issues with generators misbehaving

**Comment by bog-dan-ro** [comment]
@RubenRBS I added them after cci hook warned that I didn't have any `settings` attribute...

File: recipes/kdbindings/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self.source_folder)
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "9",
+            "Visual Studio": "15.7",
+            "clang": "7",
+            "apple-clang": "11",
```
Comment (SpaceIm) [inline recipes/kdbindings/all/conanfile.py:31]: `"msvc": "191"` should be added to this dictionary

**Pull request merged**